rules-reviewed/azure/eap/eap-to-azure-appservice-pom.windup.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+
+<ruleset id="eap-to-azure-appservice-pom"
+         xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            Show how to deploy an application to JBoss on App Service
+        </description>
+        <dependencies>
+             <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="eap" versionRange="[7,8)" />
+        <targetTechnology id="azure-appservice"/>
+        <tag>Azure</tag>
+    </metadata>
+    <rules>
+        <rule id="eap-to-azure-appservice-pom-001">
+            <when>
+                <file filename="pom.xml" />
```

Comment:
I think it's worth investing some effort to have a more restrictive condition to identify an "EAP application" otherwise the rule will become very noisy.
An idea could be identify the usage of the `org.jboss.bom` artifact and more info available https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4/html-single/development_guide/index#manage_project_dependencies

Reply:
@mrizzi I've updated the PR to use the JBoss BOM with a specific version that would match the minimum version of JBoss offered in App Service